test/integrations/validation/_fixtures/Validator.fixtures.ts

```diff
@@ -180,6 +221,10 @@ const SCHEMAS = {
     UUID: UUID_VALIDATION,
     EMAIL: EMAIL_VALIDATION,
     ARRAY: ARRAY_VALIDATION,
+    URLNO: NO_PROTOCOL_URL_VALIDATION,
```

Comment:
I didn't know what `URLNO` meant until I read this line. Can you change the name to something more descriptive? Something like `URL_NO_PROTOCOLS` or something? Can you do the same for the other URL schema's to make them more clear?